scripts/fix_sharedComponent.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs-extra');
+
+const filePath = 'src/reactViews/app/types/sharedComponents.d.ts';
+
+fs.readFile(filePath, 'utf8', (err, data) => {
+  if (err) {
+    console.error(`Error reading file: ${err}`);
+    return;
+  }
+
+  // FIX: Removing the const keyword, the type parameter declaration becomes valid
+  const updatedContent = data.replace(`TabBarMenu<const T extends string>`, 'TabBarMenu<T extends string>');
```

Comment:
Weird, is this a problem with the webpack config or babel or something like that?

Reply:
Maybe typescript version? https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#const-type-parameters